src/ngraph/frontend/onnx_import/utils/common.hpp

```diff
@@ -115,6 +115,15 @@ namespace ngraph
                                                    std::vector<std::int64_t> axes,
                                                    std::int64_t tensor_rank);
 
+            /// \brief Return the outputs of the node as vector.
+            ///
+            /// \param[in] node            Node with multiple outputs.
+            /// \param[in] outputs_number  Number of outputs in returned vector.
+            ///
+            /// \return                    Vector of outputs of input node.
+            ngraph::NodeVector get_outputs(const std::shared_ptr<ngraph::Node>& node,
```

Comment:
It would be great if `outputs_number` was optional and the function returned all outputs by default.

Reply:
Thanks. `outputs_number` was not needed. I used `get_output_size`